src/tests/ftest/control/dmg_pool_query_ranks.py

```diff
@@ -80,58 +87,65 @@ def test_pool_query_ranks_mgmt(self):
 
         all_ranks = enabled_ranks.copy()
         self.random.shuffle(all_ranks)
-        self.log.info("Starting excluding ranks: all_ranks=%s", all_ranks)
-        for rank in all_ranks:
-            self.log.debug("Excluding rank %d", rank)
-            self.pool.exclude([rank])
-            enabled_ranks.remove(rank)
-            disabled_ranks = sorted(disabled_ranks + [rank])
-
-            self.log.debug("Checking enabled ranks state information")
-            data = self.dmg.pool_query(self.pool.identifier, show_enabled=True)
-            self.assertListEqual(
-                data['response'].get('enabled_ranks'), enabled_ranks,
-                "Invalid enabled_ranks field: want={}, got={}".format(
-                    enabled_ranks, data['response'].get('enabled_ranks')))
-            self.assertListEqual(
-                data['response'].get('disabled_ranks'), disabled_ranks,
-                "Invalid disabled_ranks field: want={}, got={}".format(
-                    disabled_ranks, data['response'].get('disabled_ranks')))
-
-            self.log.debug("Waiting for pool to be rebuild")
-            self.pool.wait_for_rebuild_to_start()
-            self.pool.wait_for_rebuild_to_end()
+        exclude_rank = all_ranks[0]
+        suspect_rank = all_ranks[1]
+        self.log.info("Starting excluding rank:%d all_ranks=%s", exclude_rank, all_ranks)
```

Comment:
> The new behavior of only stopping two ranks does not align with the test description or how it currently runs. The test starts 3 ranks (with the option to expand this by changing the `test_servers` count in the test yaml). The old code would stop all three ranks (or more if more servers were specified). Is this fully intentional?

I think test logs need be updated. We don't need stop all three ranks. test intention is to test enable, disable, suspect ranks change accordingly..




> 
> Since we're modifying this test it would be beneficial use the new log step feature. This is helpful for determining how long parts of the test take and provides a unique search string to jump to in the log:

Yup, that is good idea indeed.